sdk/core/core-util/review/core-util.api.md

```diff
@@ -4,14 +4,21 @@
 
 ```ts
 
+import { AbortSignalLike } from '@azure/abort-controller';
+
 // @public
 export function computeSha256Hash(content: string, encoding: "base64" | "hex"): Promise<string>;
 
 // @public
 export function computeSha256Hmac(key: string, stringToSign: string, encoding: "base64" | "hex"): Promise<string>;
 
-// @public
-export function delay(timeInMs: number): Promise<void>;
+// Warning: (ae-forgotten-export) The symbol "DelayOptions" needs to be exported by the entry point index.d.ts
+//
+// @public (undocumented)
+export function delay<T>(timeInMs: number, options?: DelayOptions): Promise<T | void>;
+
+// @public (undocumented)
+export function delay<T>(timeInMs: number, value: T, options?: DelayOptions): Promise<T | void>;
```

Comment:
Why do we have two?

Reply:
Many calls for the `delay` method have the `value `parameter set to undefined. We want to overload the method so that the users do not have to pass in `undefined`

Reply:
I looked into this with @minhanh-phan offline, and it looks like the overload with value is not really used anywhere, so just keeping a single signature with no `value: T`